src/cascadia/TerminalControl/TermControl.h

```diff
@@ -32,6 +32,12 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
     struct TermControl : TermControlT<TermControl>
     {
+        using TimeStamp = uint64_t;
+
+        // represents 0.5 seconds (or 500 ms). Denoted in microseconds
+        // Used for PointerPoint.Timestamp Property (https://docs.microsoft.com/en-us/uwp/api/windows.ui.input.pointerpoint.timestamp#Windows_UI_Input_PointerPoint_Timestamp)
+        const TimeStamp multiClickTimer = 500000;

```

Comment:
note: fall back to the windows systemwide double click value; remove hardcoded constant

Reply:
also technically 0.5sec is not a timestamp, it's a duration